src/hooks/download.js

```diff
@@ -8,15 +8,28 @@ const getOS = (name) => {
 };
 
 const getBit = (name) => {
-  if (name.includes('x64')) return '(Intel 64-bit)';
-  else if (name.includes('windows64')) return '(64-bit)';
-  else if (name.includes('windows32')) return '(32-bit)';
-  else if (name.includes('macos-aarch64')) return '(Apple Silicon)';
-  else if (name.includes('linux-arm32')) return '(Raspberry Pi 32-bit)';
-  else if (name.includes('linux-arm64')) return '(Raspberry Pi 64-bit)';
+  if (name.includes('x64')) return 'Intel 64-bit';
```

Comment:
Not sure if this is the place to change it but the order of the buttons for Linux should be as follow: Intel 64 at the top, then Rpi 32, then Rpi 64